.codespell.ignore.txt

```diff
@@ -0,0 +1,10 @@
+geting
```

Comment:
This is a weird one, what context are we using this in?

Reply:
It's for "GETing", the ignore file being all lowercase makes it harder to spot unfortunately. Without the ignore it tries to flag all instances of "GETing" as typos of "getting".

Reply:
E.g. 

test/load-generator/boulder-calls.go
239:// getAuthorization fetches an authorization by GETing the provided URL.

Reply:
ahhh, that makes sense.